### DISCUSSION TITLE
Identify which letters the user has trouble with

### DISCUSSION THREAD

**Discussion by Kelvets** [monkeytypegame/monkeytype#2336] [Ideas]
When the user has done a lot of tests, it would be interesting if MonkeyType identified that certain letters are typed slower than others on average, or that words that contain certain letters or letter combinations have a higher error rate than average, so that the user can know where to focus to improve.

**Comment by tajhans** [comment]
I agree but this has already been mentioned here: #44 and here: #2271.
